views/manage/index.tt

```diff
@@ -0,0 +1,305 @@
+[%# manage/index.tt
+
+Authors:
+    Jen Griffin <kareila@livejournal.com>
+
+Copyright (c) 2017 by Dreamwidth Studios, LLC.
+
+This program is free software; you may redistribute it and/or modify it under
+the same terms as Perl itself.  For a copy of the license, please reference
+'perldoc perlartistic' or 'perldoc perlgpl'.
+
+%]
+
+[%- sections.title = '.title' | ml -%]
+
+[%- sections.head = BLOCK %]
+    <style type="text/css">
+        .linklist {
+          margin-left: 2em;
+        }
+        .linklist ul {
+          padding-left: 2em;
+          margin-bottom: 0.75em;
+          font-weight: bold;
+        }
+        ul.accountinfo {
+          list-style-type: none;
+        }
+        ul.accountinfo li {
+          padding-bottom: 0.5em;
+        }
+        ul.accountinfo li b {
+          padding-right: 0.5em;
+        }
+        .left {
+          float: left;
+        }
+        .right {
+          float: right;
+        }
+        .column {
+          width: 48%;
+          padding: 0.5em;
+        }
+    </style>
+[% END %]
+
+<form method="get">[% authas_html %]</form>
+
+[% iscomm = u.is_community ? '.comm' : '' %]
+
+<h1>[% '.youraccount.header' _ iscomm | ml %]</h1>
```

Comment:
Because `.` is reserved for object methods (in lieu of `->`)

I read somewhere (probably TT docs) that Perl 6 also decided to make the same change.